floodviz/map_utils.py

```diff
@@ -20,8 +38,19 @@ def site_dict(site_list, url_prefix):
     # get data from url
     req = requests.get(url)
 
+    if not req.text.startswith('#'):
```

Comment:
This is not necessary. It's possible that an rdb file would contain no comment lines.